proto/plan.proto

```diff
@@ -350,6 +350,18 @@ message AnalyzeInfo {
     int64 memory_size = 6;
 }
 
+message ExplainOption {
```

Comment:
Why do we need this?   explain option should be handled in parse level and the command run in explain mode, with stats.   But wy TEXT/JSON/DOT need to change plan proto def?

Reply:
OK, you are right. I have confirmed to Mo Chen that these are really not needed.